v1/proto/service/gribi.proto

```diff
@@ -421,3 +427,49 @@ message GetResponse {
   // A set of entries that are installed on the device.
   repeated AFTEntry entry = 1;
 }
+
+// FlushRequest is the request sent from the client to the server when a Flush
+// RPC is called.
+message FlushRequest {
+  // election specifies the election ID from a client. It is valid only
+  // when the server has clients that are operating in the SINGLE_PRIMARY mode.
+  oneof election {
+    // id indicates that the Flush ID supplied should be compared to the
+    // existing election ID on the server from the current master.
+    Uint128 id = 1;
+    // override indicates that the client explicitly requests that the election
+    // ID should not be compared, regardless of whether there are SINGLE_PRIMARY
+    // clients on the server.
+    bool override = 2;
+  }
+  // network_instance specifies the Network Instance that should be flushed.
+  oneof network_instance {
+    // The network instance from which the entries should be retrieved.
+    // If name is specified the empty string "" is considered invalid.
+    string name = 3;
+    // all indicates that entries for all network instances for the
+    // specified network instance should be returned.
+    Empty all = 4;
```

Comment:
copy paste error on the comments (`should be retrieved`/ `returned` needs to be `flushed`)

Reply:
Thanks, fixed.